### DISCUSSION TITLE
iStoreOS 24.10 测试版

### DISCUSSION THREAD

**Discussion by screousi0103** [istoreos/istoreos#1995] [Q&A] [answered]

**Comment by jjm2473** [comment]
才几天又rc3了，等正式版出来吧，才没那么多变更

**Reply by screousi0103** [reply]
首先感谢开发者回复

> 才几天又rc3了，等正式版出来吧，才没那么多变更

所以我建议了

> 可以考虑暂不发布正式版先放 [alpha](https://fw.koolcenter.com/iStoreOS/alpha/)

先出测试版，大家找 BUG，同时

1. 现在 istoreOS 是 22.03，应该也无法官方支持升级 24.10，所以要升级就代表全新安新装系统
> Sysupgrade can be used to upgrade a device from 23.05 to 24.10, and configuration will be preserved in most cases.
> - Sysupgrade from 22.03 to 24.10 is not officially supported.

2. 现在的是用的 fw3 版 22.03 ，之前有说要换 fw4，这亦是一个属于 istoreos 的 Release Candidate

3. RC 版本是已经确定大框架的 Release 版，现在出测试版可以分担一些压力

以上只是我的一些想法
最终以开发者的想法为准

**Reply by jjm2473** [reply]
openwrt rc版的内核模块是会变的，也就是过段时间可能就没法从线上安装内核模块了。而且rc本来就可能很多问题，测出的问题可能是openwrt本身就有的，昨天又出rc4了，还是等openwrt发布release再移植

**Reply by screousi0103** [reply]
>等openwrt发布release再移植

我支持开发者的决定，但我还是想说明一个误区
>openwrt rc版的内核模块是会变的，也就是过段时间可能就没法从线上安装内核模块了。

Release Candidate (RC) 也属于 Release，所以我才会在 RC 发布后提出跟进请求

参考 23.05，RC版本和正式版一致，同版本的内核不变

**Reply by jjm2473** [reply]
奇怪，24.10-rc1现在安装不了内核模块了

**Reply by screousi0103** [reply]
额，是bug，所以rc1仅两天就发rc2了。。。

https://openwrt.org/releases/24.10/notes-24.10.0-rc2#changes_between_openwrt_24100-rc1_and_24100-rc2
> # Changes between OpenWrt 24.10.0-rc1 and 24.10.0-rc2
> - Fixed opkg installation of kernel modules. The path to the kernel module packages was fixed.

**Comment by Potato4096** [comment]
同时希望官方支持树莓派5，毕竟树莓派5的性能比树莓派4强了不少，买完了才发现不支持，只能装原版，不太舒服

**Reply by jjm2473** [reply]
嗯，openwrt 24.10本身就支持rpi5，很容易加上

**Comment by jjm2473** [comment, marked as answer]
测试版本更新和反馈集中在 https://github.com/istoreos/istoreos/discussions/2228

**Reply by screousi0103** [reply]
感谢开发者制作测试版🙏

**Comment by liweilin999** [comment]
问题反馈，istoreos最新测试版24.10.0-rc4，内核6.67，istoreos-24.10.0-rc4-2025011019-x86-64-squashfs-combined-efi.img，安装emby无法识别核显，求解决，机器是n305，安装istoreos商店里emby插件也是不行，无法识别核显，然后自己创建容器也映射了核显驱动也不行，以下是创建命令
docker run -d \
--privileged \
--network=host \
-v /opt/emby2/config:/config \
-v /opt/media/:/media \
-e TZ="Asia/Shanghai" \
--device=/dev/dri:/dev/dri \
-e UID=0 \
-e GID=0 \
-e GIDLIST=022 \
--restart=unless-stopped \
--name emby \
amilys/embyserver:latest
![屏幕截图 2025-01-12 143124](https://github.com/user-attachments/assets/ac46f2a9-f1a8-4f70-abc8-e2f55bbee042)

**Comment by jjm2473** [comment]
目前已知问题：
1. Ventoy无法启动OpenWRT 24.10.0-rc4，同样也无法启动iStoreOS 24.10.0-rc4

**Reply by jjm2473** [reply]
固件更新了，这个问题已经解决。此外i915驱动支持了硬编码，低功耗编码（需硬件支持）。OAF也加上了。

**Comment by liweilin999** [comment]
大佬，刚刚试了下物理机安装确实解决了，大佬牛逼



---原始邮件---
发件人: "Liangbin ***@***.***&gt;
发送时间: 2025年1月14日(周二) 晚上10:57
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [istoreos/istoreos] iStoreOS 24.10 测试版 (Discussion #1995)




 
固件更新了，这个问题已经解决。此外i915驱动支持了硬编码，低功耗编码（需硬件支持）。OAF也加上了。

**Comment by breeze303** [comment]
24.10稳定版已发布三周，istoreos貌似进展缓慢，我试图merge到24.10最新commit但会导致luci混乱

**Reply by baihaoyun** [reply]
好久不更新了，有些插件也不能兼容，两难中

**Comment by biubiushaka** [comment]
OpenWrt 24.10稳定版发布已经一个多月了，iStoreOS这边啥时候发布稳定版呀？用的一些第三方组件的新版本已经不针对22.x维护了 :( 哭唧唧

**Reply by sanbrother** [reply]
> OpenWrt 24.10稳定版发布已经一个多月了，iStoreOS这边啥时候发布稳定版呀？用的一些第三方组件的新版本已经不针对22.x维护了 :( 哭唧唧

我试了OWRT 24.10，reboot后，系统就起不来了，NANOPI R4S

**Comment by sanbrother** [comment]
想在R4S上测试一下呢，但是好像没有版本呢，期待
